docs/contributing-documentation.md

```diff
@@ -1,54 +1,99 @@
-# Contributing documentation
+# Contributing to the HTML spec
 
 This document contains information about contributing to the HTML specification.
 
-## Making a contribution
+## Copyright and Patent Licensing
 
-The HTML specification is governed by the [W3C Patent Policy](http://www.w3.org/Consortium/Patent-Policy-20040205/), and [Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software). This ensures that the HTML standard will remain royalty free for everyone to use.
+HTML is published under the [W3C Software and Document License](http://www.w3.org/Consortium/Legal/copyright-software).
+Please do not make a Pull Request to commit content if you cannot provide it under these terms.
```

Comment:
Think warning against this at this point (without the information given below) is redundant.